public/scripts/google-analytics.js

```diff
@@ -0,0 +1,7 @@
+window.dataLayer = window.dataLayer || []
+function gtag() {
+  dataLayer.push(arguments)
+}
+gtag('js', new Date())
+
+gtag('config', 'G-WVH440CNZ3')
```

Comment:
GoogleId and TwitterId may need to be injected through environment variables instead of hardcoding them.

Reply:
I'm not sure if it's necessary to extract them to env because these code snippets are copy&paste from the platforms directly. It might be easy to leave them intact. Say, when the script should be upgraded, it can be replaced by the new code snippet directly.